src/databricks/labs/ucx/source_code/queries.py

```diff
@@ -11,6 +11,9 @@ class FromTable(Linter, Fixer):
     def __init__(self, index: MigrationIndex):
         self._index = index
 
+    def index(self):
```

Comment:
let's keep the dependencies to be injectable via constructor only, except a couple of predefined places, otherwise it may create unnecessary coupling

Reply:
done